src/storage/txn/commands/flashback_to_version.rs

```diff
@@ -48,7 +48,7 @@ impl CommandExt for FlashbackToVersion {
             FlashbackToVersionState::RollbackLock { key_locks, .. } => {
                 latch::Lock::new(key_locks.iter().map(|(key, _)| key))
             }
-            FlashbackToVersionState::Prewrite { key_to_lock } => latch::Lock::new([key_to_lock]),
+            FlashbackToVersionState::Prewrite { start_key } => latch::Lock::new([start_key]),
```

Comment:
Emm, the key locked in the latch is different from the actual written key...

As there cannot be any other valid write requests, it may be safe... But is it possible to make it correct by reading the key to lock in the read phase? I will be less worried :)

Reply:
Thx for ur idea! Had to address it, PTAL!